bundles/org.openhab.binding.guntamatic/README.md

```diff
@@ -76,122 +77,123 @@ The Guntamatic Heating System can be controlled using the following channels:
-| `bufferTop0`         | Buffer Top 0           | `Number:Temperature`   | `°C` |        🔐 W1         |   R/O    |  false   |
-| `bufferBtm0`         | Buffer Btm 0           | `Number:Temperature`   | `°C` |        🔐 W1         |   R/O    |  false   |
-| `bufferTop1`         | Buffer Top 1           | `Number:Temperature`   | `°C` |        🔐 W1         |   R/O    |  false   |
-| `bufferBtm1`         | Buffer Btm 1           | `Number:Temperature`   | `°C` |        🔐 W1         |   R/O    |  false   |
-| `bufferTop2`         | Buffer Top 2           | `Number:Temperature`   | `°C` |        🔐 W1         |   R/O    |  false   |
-| `bufferBtm2`         | Buffer Btm 2           | `Number:Temperature`   | `°C` |        🔐 W1         |   R/O    |  false   |
-| `bExtraWw0`          | B extra-WW. 0          | `Switch`               |      |        🔐 W1         |   R/O    |  false   |
-| `bExtraWw1`          | B extra-WW. 1          | `Switch`               |      |        🔐 W1         |   R/O    |  false   |
-| `bExtraWw2`          | B extra-WW. 2          | `Switch`               |      |        🔐 W1         |   R/O    |  false   |
-| `auxiliaryPump0`     | Auxiliary pump 0       | `Switch`               |      |        🔐 W1         |   R/O    |  false   |
-| `auxiliaryPump1`     | Auxiliary pump 1       | `Switch`               |      |        🔐 W1         |   R/O    |  false   |
-| `auxiliaryPump2`     | Auxiliary pump 2       | `Switch`               |      |        🔐 W1         |   R/O    |  false   |
-| `boilersConditionNo` | Boiler´s condition no. | `String`               |      |        🔐 W1         |   R/O    |  false   |
-| `bufferT5`           | Buffer T5              | `Number:Temperature`   | `°C` |        🔓 W0         |   R/O    |  false   |
-| `bufferT6`           | Buffer T6              | `Number:Temperature`   | `°C` |        🔓 W0         |   R/O    |  false   |
-| `bufferT7`           | Buffer T7              | `Number:Temperature`   | `°C` |        🔓 W0         |   R/O    |  false   |
-| `extraWw0`           | Extra-WW. 0            | `Number:Temperature`   | `°C` |        🔓 W0         |   R/O    |  false   |
-| `extraWw1`           | Extra-WW. 1            | `Number:Temperature`   | `°C` |        🔓 W0         |   R/O    |  false   |
-| `extraWw2`           | Extra-WW. 2            | `Number:Temperature`   | `°C` |        🔓 W0         |   R/O    |  false   |
-| `grate`              | Grate                  | `Number:Dimensionless` | `%`  |        🔓 W0         |   R/O    |  false   |
+| Channel                  | Description            | Type                   | Unit | Security Access Level | ReadOnly | Advanced |
+| ------------------------ | ---------------------- | ---------------------- | :--: | :-------------------: | :------: | -------: |
+| `000_running`            | Running                | `String`               |      | 🔓 W0                 | R/O      | false    |
```

Comment:
rationale is documented here:
https://community.openhab.org/t/guntamatic-new-binding-for-guntamatic-heating-systems-biostar-powerchip-powercorn-biocom-pro-therm-biosmart/128451/46

Binding reads data with unknown description from heating system and creates one channel per data element. The description is used to define the channel name. It turned out, that on some devices it happens that several data elements share the same description. The channel Id was no longer unique.
All the data elements come already with an index (which is unique). The new implementation makes advantage of the unique index.

Alternative solution: Check if the channel name was already used and add a postfix only to these channels. I think the implementation of this PR is more straight forward.

As due to the introduction of groups, the channel name anyhow breaks, I think this change should be acceptable. Anyhow it will come for the big OH5.0.0 release ...

Reply:
done